sdk/servicebus/service-bus/test/internal/receiver.spec.ts

```diff
@@ -240,4 +243,65 @@ describe("Receiver unit tests", () => {
       return sub;
     }
   });
+
+  describe("getMessageIterator", () => {
+    it("abortSignal is passed through (receiver)", async () => {
+      const impl = new ServiceBusReceiverImpl(
+        createConnectionContextForTests(),
+        "entity path",
+        "peekLock"
+      );
+
+      const abortSignal = createAbortSignalForTest(true);
+
+      try {
+        const iter = impl.getMessageIterator({
+          abortSignal
+        });
+
+        await iter.next();
+        assert.fail("Should have thrown");
+      } catch (err) {
+        assert.equal("AbortError", err.name);
```

Comment:
```suggestion
        assert.equal(err.name, "AbortError");
```
I think... 1st - actual, 2nd - expected

Reply:
You're totally right.